### PR TITLE
Restore old release process for older branches

### PR DESCRIPTION
**Pull request by kargakis**
@stevekuznetsov I think this is the best approach out of the three. Testing https://github.com/openshift/origin/pull/14422 in https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_kargakis/1/console

### CODE REVIEW

File: sjb/config/common/test_cases/origin_release.yml [hunk 1]
```diff
@@ -6,6 +6,17 @@ extensions:
       title: "build an origin release"
       repository: "origin"
       script: |-
+        CURRENT_BRANCH=$(git rev-parse --abbrev-ref --symbolic-full-name HEAD)
```
Comment (stevekuznetsov) [inline sjb/config/common/test_cases/origin_release.yml]: Use `$ORIGIN_TARGET_BRANCH`

Comment (kargakis) [inline sjb/config/common/test_cases/origin_release.yml]: done

**Comment by kargakis** [comment]
The build in https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_kargakis/1/console was successful. @stevekuznetsov ok to merge and roll out?

File: sjb/config/common/test_cases/origin_release.yml [hunk 2]
```diff
@@ -6,6 +6,17 @@ extensions:
       title: "build an origin release"
       repository: "origin"
       script: |-
+        ORIGIN_TARGET_BRANCH=$(git rev-parse --abbrev-ref --symbolic-full-name HEAD)
```
Comment (stevekuznetsov) [inline sjb/config/common/test_cases/origin_release.yml]: Sorry, I meant that there is a variable already available to you -- `$<REPO_NAME>_TARGET_BRANCH` for whatever is being tested. No need to do this.

Comment (kargakis) [inline sjb/config/common/test_cases/origin_release.yml]: ok, updated once more

**Comment by kargakis** [comment]
Kicked https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_kargakis/2/console - will merge and push to Jenkins once the build passes.

File: sjb/config/common/test_cases/origin_release.yml (hunk 2 above)
Comment (kargakis) [inline sjb/config/common/test_cases/origin_release.yml]: https://github.com/openshift/aos-cd-jobs/issues/315

**Pull request merged**